frontend/src/plugins/i18n-messages.ts

```diff
@@ -0,0 +1,28 @@
+import { merge } from "lodash-es";
+
+const localPathPrefix = "../locales/";
+
+// import i18n resources
+// https://vitejs.dev/guide/features.html#glob-import
+export const mergedLocalMessage = Object.entries(
+  import.meta.glob("../locales/**/*.json", { eager: true })
```

Comment:
```suggestion
  import.meta.glob(`${localPathPrefix}**/*.json`, { eager: true })
```

Reply:
Sadly we cannot use variable in `import.meta.glob` :(

```
Invalid glob import syntax: Expected glob to be a string, but got dynamic template literal
```